django_pgjsonb/fields.py

```diff
@@ -62,7 +62,15 @@ def get_prep_lookup(self, lookup_type, value, prepared=False):
             # This will cast numbers to strings, but also grab the keys
             # from a dict.
             value = ['%s' % v for v in value]
-
+        if lookup_type == 'near':
+            # geo type must have 3 item, longitude, latitude and search
+            # range, allowed_format 'lat,lng,rang' or [lat, lng, rang]
+            if isinstance(value, six.string_types):
+                value = value.split(',')
+            # if parmas not verify, just don't use this filter, 12756000
+            # is the farest long in earth
+            if len(value) != 3:
+                value = [0, 0, 12756000]
```

Comment:
I tend to raise an error if the input not match the format it required, otherwise it's just quietly ignore the error which would be make trouble to debug. would you agree?